Guesser/FilterTypeGuesser.php

```diff
@@ -71,7 +71,7 @@ public function guessType($class, $property, ModelManagerInterface $modelManager
                 $options['field_type']    = 'sonata_type_boolean';
                 $options['field_options'] = array();
 
-                return new TypeGuess('sonata_search_elastic_boolean', $options, Guess::HIGH_CONFIDENCE);
+                return new TypeGuess('sonata_search_elastica_boolean', $options, Guess::HIGH_CONFIDENCE);
```

Comment:
Sure, you can check under services declaration, it's a filter type:
```xml
        <service
            id="sonata.admin.search.filter.type.boolean"
            class="Sonata\AdminSearchBundle\Filter\BooleanFilter">
            <tag name="sonata.admin.filter.type" alias="sonata_search_elastica_boolean" />
        </service>
```
Comes from https://github.com/ruflin/Elastica